patroni/postgresql/callback_executor.py

```diff
@@ -1,40 +1,44 @@
-import logging
-import subprocess
 from threading import Event, Lock, Thread
+import subprocess
+import logging
+import time
+import queue
```

Comment:
It breaks compatibility with python 2.7, therefore all tests are [failing](https://travis-ci.org/zalando/patroni/builds/599792802?utm_source=github_status&utm_medium=notification).
In order to remain compatible you have to use `six` to import `Queue`.